src/Components/M_schedule/M_schedule.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { BiCategory } from "react-icons/bi";
+import info from "../../assets/icons/info.svg";
```

Comment:
This line is flagging an error.
This is because the file name is `Info.svg` and you call it here as`info.svg`. Some systems will flag errors for case sensitivity. Kindly fix this.